src/test/java/com/virtuslab/archunit/ForbiddenMethodsTestSuite.java

```diff
@@ -173,4 +174,18 @@ public void no_classes_should_call_println() {
         .should().callMethodWhere(name("println"))
         .check(importedClasses);
   }
+
+  // Note that https://checkstyle.sourceforge.io/config_coding.html#CovariantEquals doesn't cover methods defined in interfaces.
```

Comment:
do they have an issue for it? if not it may be worth to open one

Reply:
Couldn't find anything existing, opened https://github.com/checkstyle/checkstyle/issues/12298